content/en/agent/logs/advanced_log_collection.md

```diff
@@ -58,13 +60,78 @@ logs:
 ```
 
 {{% /tab %}}
-{{% tab "include_at_match" %}}
+{{% tab "Docker" %}}
+
+In a Docker environment, use the label `com.datadoghq.ad.logs` on your container to specify the `log_processing_rules`, for example:
+
+```yaml
+ labels:
+    com.datadoghq.ad.logs: >-
+      [{
+        "source": "java",
+        "service": "cardpayment",
+        "log_processing_rules": [{
+          "type": "exclude_at_match",
+          "name": "exclude_datadoghq_users",
+          "pattern" : "\\w+@datadoghq.com"
+        }]
+      }]
+```
+
+**Note**: Escape regex characters in your patterns when using labels. For example, `\d` becomes `\\d`, `\w` becomes `\\w`, etc.
+
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+In a Kubernetes environment, use the pod annotation `ad.datadoghq.com` on your pod to specify the `log_processing_rules`, for example:
+
+```yaml
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  name: cardpayment
+spec:
+  selector:
+    matchLabels:
+      app: cardpayment
+  template:
+    metadata:
+      annotations:
+        ad.datadoghq.com/cardpayment.logs: >-
+          [{
+            "source": "java",
+            "service": "cardpayment",
+            "log_processing_rules": [{
+              "type": "exclude_at_match",
+              "name": "exclude_datadoghq_users",
+              "pattern" : "\\w+@datadoghq.com"
+            }]
+          }]
+      labels:
+        app: cardpayment
+      name: cardpayment
+    spec:
+      containers:
+        - name: cardpayment
+          image: cardpayment:latest
+```
+
+**Note**: Escape regex characters in your patterns when using pod annotations. For example, `\d` becomes `\\d`, `\w` becomes `\\w`, etc.
+
+{{% /tab %}}
+{{< /tabs >}}
+
```

Comment:
Add a link to the FAQ/guide to So the the user can have more example of classique scrubbing rules.

Reply:
link is in its own section already on the page :)